src/store/student-store.js

```diff
@@ -42,11 +49,26 @@ class StudentStore extends Store {
             return item.timestamp < now;
         });
     }
+    isCurrentStudent(student){
+        return student.status === activeStatus;
+    }
+
+    isNewStudent(student) {
+        let studentArray = Object.keys(students).map(key => students[key]);
+        return studentArray.filter(s => s.id == student.id).length === 0;
+    }
 
 }
 
 const store = new StudentStore();
 
+function updateStudent(id, name, email, major){
+    let student = store.getStudentById(id);
```

Comment:
NACK: You shouldn't be able to modify the store's state with one of the store's public methods. Get the student directly from `students`. Isn't it just `students[id]`?